recipes/highered/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "highered" %}
+{% set version = "0.2.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/highered-{{ version }}.tar.gz
+  sha256: 5fcae90599dda98560d6f5347ff88aee56a4b8e7181d009852c3dc699c336fb7
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - numpy
+    - pyhacrf-datamade >=0.2.0
+    - python
```

Need to specify a Python version range for noarch:
```suggestion
  host:
    - pip
    - python >=3.6
  run:
    - numpy
    - pyhacrf-datamade >=0.2.0
    - python >=3.6
```